deploy/packaging/redhat/redhat_build_rpms.py

```diff
@@ -92,15 +92,28 @@ def doRequire(info, out, root, require):
         common.writeb(out, "Requires: mdsplus%(bname)s-%(reqpkg)s = %(major)d.%(minor)d-%(release)d.%(dist)s\n" % info)
 
 
+def checkSigningKey():
+    try:
+        os.stat('/sign_keys/.gnupg')
+        return True
+    except:
+        return False
+
+
 def build():
+
+    has_key = checkSigningKey()
```

Comment:
You might simplify this with `os.path.exists`

Reply:
Yes, of course.   That is also a good catch.   I'll make the change.

Reply:
Change made.